### PR TITLE
fixing leftover persisted values in numval and integratedoffer Servic…

### PR DESCRIPTION
**Pull request by DominicFath**
…e - fixing extracost duplication for rent

### CODE REVIEW

File: src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php [hunk 1]
```diff
@@ -453,6 +427,36 @@ private function getCalculatedPrices($type = 'rent', $currency) {
 
                 $nullcheck = array_merge($nullcheck, $nullcheck_addition);
             }
+
```
Comment (josias-r) [inline src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php:430]: Why move this down? :D

File: src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php [hunk 2]
```diff
@@ -453,6 +427,36 @@ private function getCalculatedPrices($type = 'rent', $currency) {
...
+            $extraCosts = null;
+            if (isset($this->property['_embedded']['extracosts'])) {
+                foreach ($this->property['_embedded']['extracosts'] as $extracost) {
```
Comment (josias-r) [inline src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php:433]: Why move this down? :D

File: src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php [hunk 3]
```diff
@@ -524,7 +528,7 @@ private function getCalculatedPrices($type = 'rent', $currency) {
             //   ['priceNettoPerSqmPerYear', 'renders'],
             //   ['priceNettoPerTotalPerMonth', 'renders'],
             //   ['priceNettoPerTotalPerYear', 'renders'],
-            ['extraCosts', 'special'],
```
Comment (josias-r) [inline src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php:527]: Did you remove this, mecause we are now always using extraCostsPerMonth?

File: src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php (hunk 2 above)
Comment (DominicFath) [reply src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php:433]: @josias-r that it keeps the same order than it has now on iCasa :)

File: src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php (hunk 3 above)
Comment (DominicFath) [reply src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php:527]: @josias-r yes correct otherwise it'll be double and on rent - but yes now it'll be limited to the monthly calculation...
maybe it would make sense to adjust the logic and make it dependable on if monthly or yearly?

Comment (josias-r) [reply src/CasasoftStandards/src/CasasoftStandards/Service/ConversionService.php:527]: Hmm I see... Well from the cassone side, the values are always displayed as "monthly" (rental).
I think that's why I added it in the first place, so all values are consistently formatted.

TLDR casaone won't make use of the yearly value, so we don't need it for that sake, idk about other services
